src/gsy_e/setup/tekniker_cop_model/README.md

```diff
@@ -0,0 +1,67 @@
+# Individual COP model integration
+
+This document describes to train COP models, developed by [Tekniker](https://www.tekniker.es/en/) by deploying the [DOE-2](https://www.doe2.com/) building energy analysis program, to yield a statistical model by leveraging measurements of select heat pump manufacturers under predefined operational conditions.
+
+## COP model training
+
+The goal of training the COP model is to produce a JSON file that contains the COP model parameters for a specific heat pump.
+For training the COP model, performance data for the selected heat pump is needed, that comply to the characteristics listed in the following section.
+
+### Heat Pump performance data characteristics
+#### Heat production data
+
+A table of heat production (Q) in kW as a function of air temperature and condensor temperature needs to be provided for the full load operation of the heat pump.
+The temperature range that is provided, is the range that the COP model will produce valid data.
+Consequently, a large range and a high resolution of input data will result in more realistic output values.
+In the following, an example table is shown.
```

Comment:
I would add the sentence:
> In the following, an example table is shown.
as a legend of the table, named "Example heat production table"